packages/inter-protocol/test/psm/setupPsm.js

```diff
@@ -0,0 +1,184 @@
+  produce.chainStorage.resolve(makeMockChainStorageRoot());
+  produce.board.resolve(makeBoard());
+
+  return { produce, consume, ...spaces };
+};
+
+/**
+ * @param {*} t
+ * @param {{ committeeName: string, committeeSize: number}} electorateTerms
+ * @param {ManualTimer | undefined=} timer
+ * @param {FarZoeKit} [farZoeKit]
+ */
+export const setupPsm = async (
+  t,
+  electorateTerms = { committeeName: 'The Cabal', committeeSize: 1 },
+  timer = buildManualTimer(t.log),
+  farZoeKit,
+) => {
+  if (!farZoeKit) {
+    farZoeKit = await setUpZoeForTest();
+  }
+
+  const knutIssuer = makeIssuerKit('KNUT');
+  const { feeMintAccess, zoe } = farZoeKit;
+  const space = await setupPsmBootstrap(timer, farZoeKit);
+  space.produce.zoe.resolve(farZoeKit.zoe);
+  space.produce.feeMintAccess.resolve(feeMintAccess);
+  const { consume, brand, issuer, installation, instance } = space;
+  const psmBundle = await provideBundle(t, psmRoot, 'psm');
+  installation.produce.psm.resolve(E(zoe).install(psmBundle));
+
+  brand.produce.AUSD.resolve(knutIssuer.brand);
+  issuer.produce.AUSD.resolve(knutIssuer.issuer);
+
+  const istIssuer = await E(zoe).getFeeIssuer();
+  const istBrand = await E(istIssuer).getBrand();
+
+  brand.produce.IST.resolve(istBrand);
+  issuer.produce.IST.resolve(istIssuer);
+
+  await Promise.all([
+    startEconomicCommittee(space, {
+      options: { econCommitteeOptions: electorateTerms },
+    }),
+    startPSM(space, {
+      options: {
+        anchorOptions: {
+          denom: 'AUSD',
+          decimalPlaces: 6,
+          keyword: 'AUSD',
+          proposedName: 'AUSD',
+        },
+      },
+    }),
+  ]);
+
+  const installs = await allValues({
+    psm: installation.consume.psm,
+    governor: installation.consume.contractGovernor,
+    electorate: installation.consume.committee,
+    counter: installation.consume.binaryVoteCounter,
+  });
+
+  const governorCreatorFacet = consume.psmGovernorCreatorFacet;
+  const governorInstance = await instance.consume.psmGovernor;
+  const governorPublicFacet = await E(zoe).getPublicFacet(governorInstance);
+  const g = {
+    governorInstance,
+    governorPublicFacet,
+    governorCreatorFacet,
+  };
+  const governedInstance = E(governorPublicFacet).getGovernedContract();
+
+  /** @type { GovernedPublicFacet<PSM> } */
+  const psmPublicFacet = await E(governorCreatorFacet).getPublicFacet();
+  const psm = {
+    psmCreatorFacet: await consume.psmCreatorFacet,
+    psmPublicFacet,
+    instance: governedInstance,
+  };
+
+  const committeeCreator = await consume.economicCommitteeCreatorFacet;
+  const electorateInstance = await instance.consume.economicCommittee;
+
+  const poserInvitationP = E(committeeCreator).getPoserInvitation();
+  const poserInvitationAmount = await E(
+    E(zoe).getInvitationIssuer(),
+  ).getAmountOf(poserInvitationP);
+
+  /** @type {import('@agoric/vats/tools/storage-test-utils.js').MockChainStorageRoot} */
+  // @ts-expect-error cast
```

Comment:
you could get this directly from `setupPsmBootstrap` where it has the type `MockChainStorageRoot`.

Reply:
in short, better source for the type info. If you exported the value from `setupPsmBootstrap` then the type would come along with it and you wouldn't need the annotation/cast here.

I think you _could_ call `makeMockChainStorageRoot()` again here, but I don't see a benefit. It's important that each test have its own chain storage, but each test is calling `setupPsm` and each of those is calling `setupPsmBootstrap`.

Reply:
done